recipes/default.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Cookbook Name:: w32time
+# Recipe:: default
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+# This recipe will Configure w32time service to use ntp
+
+# Make sure w32time service is running
+service node['w32time']['service'] do
+  service_name node['w32time']['service']
+  supports :status => true
+  action [ :enable, :start ]
+end
+
+# Command to configure w32time service
+# 0x8 stands for time client
+execute 'set ntp config' do
+  command "w32tm /config /update /manualpeerlist:\"#{node['w32time']['ntpserver']},0x8\" /syncfromflags:manual /reliable:yes"
```

Comment:
OPT: `#{node['w32time']['ntpserver']},0x8` is repeated twice, it might worth it to have a variable for that